data/phi0474/phi031/phi0474.phi031.perseus-lat1.xml

```diff
@@ -80,112 +86,58 @@
 		</profileDesc>
 
 		<revisionDesc>
-			<!--
-      <change><date>5/9/97</date>
-	<respStmt><name>CJSU</name><resp>ed.</resp></respStmt>
-	<item>
-$Log: cic.oct2_lat.xml,v $
-Revision 1.4  2013-09-13 11:21:40  balmas01
-reverting texts back to pre_cts_reorg tagged version
-
-Revision 1.2  2011-07-19 22:30:38  gcrane
-removed the earlier persName/placeName tags
-
-Revision 1.1  2011-01-15 23:14:25  gcrane
-base SDL file
-
-Revision 1.1  2009/10/09 19:49:13  rsingh04
-more reorganizing of texts module by collection
-
-Revision 1.5  2006/04/02 18:12:19  gcrane
-mainly fixes to the words lower cased at the start of sentences:
-found that we had always had <name>Cum</name> rather than <reg>cum</reg>.
-
-This has been fixed in oct.cic[123]
-
-oct.cic[456] are having the app crits added and will need to be updated when
-those files come back
-
-Revision 1.4  2006/03/31 17:49:52  gcrane
-missing charaters from imaging problems have been checked
-
-Revision 1.3  2006/03/29 01:03:14  gaweaver
-this includes the notes from the app crit.
-
-there is one known problem: the lower right hand corner of one page got
-clipped and the characters in the app crit need to be added with reference
-to a clean copy
-
-Revision 1.2  2005/08/01 15:59:19  packel
-removed stray item tags
-
-	Revision 1.1  2005/07/25 14:31:11  packel
-	Converted to XML
-	
-Revision 1.13  2005/04/08 17:56:07  mimno
-added default chunks
-
-Revision 1.12  2003/07/01 22:14:57  yorkc
-Updated texts to TEI P4 and Perseus P4 extensions; minor cleanup (esp. character encodings and typos.)
-
-Revision 1.11  2003/05/20 21:16:12  amahoney
-do not mark Heus as a proper name
-
-Revision 1.10  2002/12/02 18:41:45  amahoney
-correct spelling of Quirites, improperly 'regularized'
-
-Revision 1.9  2000/03/06 22:21:48  amahoney
-add headers for groups of speeches, so they show up in the toc:  Catilines,
-Verrines, Philippics, and Agrarian Law
-
-Revision 1.8  2000/03/04 20:34:10  dasmith
-Added separate funder entity to TEI header.
-
-Revision 1.7  1999/10/14 21:36:02  dasmith
-Added explanatory subtitles to titleStmt
-
-Revision 1.6  1999/08/12 17:17:11  dasmith
-Fixed refsDecl
-
-Revision 1.5  1999/08/04 18:11:59  textgod
-Random changes over a period of months.
-
-Revision 1.4  1997/08/18 14:54:26  textgod
-???
-
-Revision 1.3  1997/05/12 19:55:57  cjsu
-Fixed type attribute.
-
-Revision 1.2  1997/05/09 21:04:08  cjsu
-Fixed header info. Added abbreviations.
-
-Revision 1.1  1997/05/09 20:57:17  cjsu
-Added to repository. Passes SGML validation.
-
-
-</item></change>
-    <change><date>July 2014</date>
-    <respStmt><name>Stella Dee</name><resp>ed.</resp></respStmt>
-    <item>split composite text and converted to unicode</item>
-      </change>-->
+			<change n="1.4" when="2013-09-13T11:21:40" who="balmas01">reverting texts back to pre_cts_reorg tagged version</change>
```

Comment:
I struck the time stamp data — too much — and the n attribute?? (there are duplicates in the n attribute anyhow). Just sort newest to oldest and it will be fine.
Make sure to add your changes here.

Empty logs are not needed such as `<change n="1.4" when="1997-08-18T14:54:26" who="textgod">???</change>`